tools/dplsh/Dockerfile

```diff
@@ -3,14 +3,18 @@
 ARG AZURE_CLI_TAG=latest
 ARG DPLSH_BUILD_VERSION=latest
 
+
 # We use the official azure cli as a base-image. It is itself based on alpine
 # and is quite minimal.
 FROM mcr.microsoft.com/azure-cli:$AZURE_CLI_TAG
 ARG TASK_VERSION=v3.0.0
 ARG TERRAFORM_RELEASE=0.15.1
+ARG HELM_VERSION=canary
```

Comment:
We should probably use a stable version instead. Actually I just had an idea, push coming up....